src/Icons/src/Icon.php

```diff
@@ -189,8 +189,8 @@ public function withAttributes(array $attributes): self
                 throw new \InvalidArgumentException(\sprintf('Invalid attribute name "%s".', $name));
             }
 
-            if (!\is_string($value) && !\is_bool($value)) {
-                throw new \InvalidArgumentException(\sprintf('Invalid value type for attribute "%s". Boolean or string allowed, "%s" provided. ', $name, get_debug_type($value)));
+            if (!\is_string($value) && !\is_bool($value) && !\is_int($value) && !\is_float($value)) {
```

Comment:
Do you have any real-life usage for a float ? 
```suggestion
            if (!\is_string($value) && !\is_bool($value) && !\is_int($value)) {
```

Reply:
You need to adjust the docblock here too. And probably on the renderer. But i'd like to have more explanation than "fix" :))